brands/amenity/pub.json

```diff
@@ -6,5 +6,25 @@
       "brand": "Ezītis miglā",
       "name": "Ezītis miglā"
     }
+  },
+  "amenity/pub|Wetherspoons": {
+    "countryCodes": ["gb"],
+    "tags": {
+      "amenity": "pub",
+      "brand": "Wetherspoons",
+      "brand:wikidata": "Q6109362",
+      "brand:wikipedia": "en:J D Wetherspoon",
+      "name": "Wetherspoons"
+    }
+  },
+  "amenity/pub|Walkabout": {
+    "countryCodes": ["gb"],
+    "tags": {
+      "amenity": "pub",
+      "brand": "Walkabout",
+      "brand:wikidata": "Q7962149",
+      "brand:wikipedia": "en:Walkabout",
```

Comment:
looks like the actual wikipedia page for this should be `"en:Walkabout (pub chain)"`

Reply:
done